spatialconnect/src/main/java/com/boundlessgeo/spatialconnect/dataAdapter/GeoPackageAdapter.java

```diff
@@ -93,13 +93,16 @@ public GeoPackageAdapter(Context context, SCStoreConfig scStoreConfig) {
             @Override
             public void call(final Subscriber<? super SCDataAdapterStatus> subscriber) {
 
-                Log.d(LOG_TAG, "Connecting to " + context.getDatabasePath(scStoreConfig.getName()).getPath());
+                Log.d(LOG_TAG, "Connecting to " + context.getDatabasePath(scStoreConfig.getUniqueID()).getPath());
                 adapterInstance.setStatus(SCDataAdapterStatus.DATA_ADAPTER_CONNECTING);
 
-                if (context.getDatabasePath(scStoreConfig.getName()).exists()) {
-                    Log.d(LOG_TAG, "GeoPackage " + scStoreConfig.getName() + " already exists.  Not downloading.");
+                // The Database's name on disk is its store ID. This is to guaruntee
```

Comment:
s/guaruntee/guarantee  Also, do you think you can fit the comment into one line?